jdi-light-examples/src/main/java/io/github/epam/EpamGithubNonStaticSite.java

```diff
@@ -0,0 +1,64 @@
+package io.github.epam;
+
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.Frame;
+import com.epam.jdi.light.elements.pageobjects.annotations.JSite;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import io.github.epam.pages.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'io.github.epam.pages'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156)

Reply:
this import used for private HomePage homePage;
all imports are used